docs/protocol/js-ceramic/guides/ceramic-nodes/running-cloud.md

```diff
@@ -71,8 +71,6 @@ The Ceramic state store and IPFS datastore are stored on your machine's filesyst
 
 The fastest way to ensure data persistence is by mounting a persistent volume to your instances and configuring the Ceramic and IPFS nodes to write to the mount location. The mounted volume should be configured such that the data persists if the instance shuts down.
 
```

Comment:
Should we also update this "Data Persistence" section to mention the ComposeDB postgres database since that is also something that needs to be backed up?

Probably would also be good to call out that if you're taking a backup of all 3 layers, you should snapshot postgres first, the state store second, and the ipfs block store third.  That way we guarantee that the higher-level subsystems won't know about data that the lower-level subsystems are missing in the backup.

Reply:
Yeah I think so. By the way, I don't see that covered in the ComposeDB section either (the cadence part): https://developers.ceramic.network/docs/composedb/guides/composedb-server/running-in-the-cloud

Probably should be represented in both places?

Check out the latest commit for updated wording